README.md

```diff
@@ -1,259 +1,12 @@
-# iotics-host-lib
+## Stomp-Client
 
-Iotics host library - a library to aid integration with Iotics
+This repository contains the source for the IOTICS stomp client & generated REST client objects.
 
-This library is a Python wrapper which abstracts calls to our Python `iotic.web.rest.client`, `iotic.web.stomp` and
-`iotic.lib.identity` client libraries. Note: the REST and STOMP interfaces can instead be used directly to integrate
-with Iotics if preferred, e.g. if using another language other than Python.
+Until the upstream `stomp.py` next release, which contains a websocket implementation, is available, the IOTICS stomp client can be used instead.
```

Comment:
Could link to here:
https://github.com/jasonrbriggs/stomp.py/blob/0c9d45c6391555d1a462d1ee2cfff95d03aaa871/CHANGELOG.md?plain=1#L6